streaming/src/main/scala/org/apache/spark/streaming/receiver/Receiver.scala

```diff
@@ -242,8 +242,7 @@ abstract class Receiver[T](val storageLevel: StorageLevel) extends Serializable
   }
 
   /**
-   * Get the unique identifier the receiver input stream that this
-   * receiver is associated with.
+   * Get the unique identifier of the receiver input stream that this receiver is associated with.
```

Comment:
OK, probably not worth doing in general. There are a bunch of lines like that that could be reflowed.